azure-pipelines.yml

```diff
@@ -0,0 +1,47 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+
+trigger:
+- master
+
+jobs:
+- job: Linux
+  pool:
+    vmImage: 'ubuntu-16.04'
```

Comment:
Ubuntu 18.04.2 is the latest Ubuntu LTS. Any reason this is on 16 instead?

If not, let's switch to 18.

Reply:
Currently, we only offer 16.04 on our hosted agents. I was told that we're working on offering 18.04 and I have reached out to the people working on it to get you some information on the timeline of when this will be available.